src/ui/components/filters/filterboxcomponent.js

```diff
@@ -199,7 +199,7 @@ export default class FilterBoxComponent extends Component {
         container: `.js-yext-filterbox-filter${i}`,
         showReset: this.config.resetFilter,
         resetLabel: this.config.resetFilterLabel,
-        showExpand: this.config.expand,
+        showExpand: config.expand === undefined ? this.config.expand : config.expand,
```

Comment:
Could we not keep it showExpand in the FacetsComponent and just refer to it properly here? 

showExpand: config.showExpand === undefined ? ....

I'd rather the FacetsComponent use the correct terminology when specifying config for FilterOptions.

Reply:
The correct config option name for FilterBox is config.expand though, not showExpand

Reply:
whereas the correct config name for FilterOptions is `showExpand`

Reply:
I can't read